src/main/java/com/exactpro/th2/common/schema/grpc/configuration/GrpcEndpointConfiguration.java

```diff
@@ -16,11 +16,11 @@
 
 package com.exactpro.th2.common.schema.grpc.configuration;
 
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
 import com.exactpro.th2.common.schema.message.configuration.Configuration;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.Data;
 
-@Data
```

Comment:
Lombok `@Data` also adds equals and hashcode methods. We should check if we can live without them or add them manualy

Reply:
GrpcEndpointConfiguration is used as a map value